shipit.yml

```diff
@@ -0,0 +1,4 @@
+deploy:
```

Comment:
To publish both at the same time, it could look something like this:

```yml
dependencies:
  override:
    - yarn
    - bundle install
deploy:
  pre:
    - yarn test
    - bundle exec rake build
    # Use auto-dist-tag to still allow publishing beta, alpha, and rc versions
    - npm install -g auto-dist-tag
    - auto-dist-tag --write
  override:
    - bundle exec package_cloud push shopify/gems pkg/*.gem
    - npm publish
```